packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/short-transform.yaml

```diff
@@ -0,0 +1,12 @@
+Resources:
+  Bucket1:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName:
+        !Transform {
+          "Name": "SomeMacroName",
+          "Parameters": {
+            key1: value1,
+            key2: value2,
+          }
+        }
```

Comment:
What's with the JSON all of a sudden? I think this should be

```yaml
      BucketName:
        !Transform
          Name: SomeMacroName
          Parameters
            key1: value1
            key2: value2
```

Reply:
I thought it was weird too, but the cloudformation docs here show a json-style syntax for the short form:

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-transform.html